### PR TITLE
Event ids and more logging

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Fixes https://github.com/aspnet/Hosting/issues/363 https://github.com/aspnet/Hosting/issues/374

- Event ids to logger events
- Startup/shutdown event ids

Please review @Tratcher @rynowak

### CODE REVIEW

**Comment by Tratcher** [comment]
LGTM, @rynowak?

File: src/Microsoft.AspNet.Hosting/Internal/HostingLoggerExtensions.cs [hunk 1]
```diff
@@ -50,13 +50,51 @@ public static void RequestFailed(this ILogger logger, HttpContext httpContext, i
...
+        }
+
+        public static void Shutdown(this ILogger logger)
+        {
+            if (logger.IsEnabled(LogLevel.Information))
+            {
+                logger.LogInformation(
+                    eventId: LoggerEventIds.Shutdown,
+                    message: "Hosting shutdown");
+            }
```
Comment (rynowak) [inline src/Microsoft.AspNet.Hosting/Internal/HostingLoggerExtensions.cs]: None of these should be information

**Comment by victorhurdugaci** [comment]
Updated

**Comment by rynowak** [comment]
:shipit:

**Pull request merged**